Makefile

```
@@ -0,0 +1,10 @@
+PODSPEC = swift-user-defaults.podspec
+
+setup:
+	bundle install
+
+lint:	setup
+	bundle exec pod lib lint --allow-warnings
+
+release: lint
+	bundle exec pod trunk push $(PODSPEC)
```

We can avoid `$(PODSPEC)` since the repo only has one and `pod` is smart enough to detect it 🙂 

```suggestion
	bundle exec pod trunk push
```